PersonListener/PersonListener.csproj

```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="AWSXRayRecorder.Handlers.AwsSdk" Version="2.8.3" />
     <PackageReference Include="Hackney.Core.DynamoDb" Version="1.30.0" />
     <PackageReference Include="Hackney.Core.Logging" Version="1.30.0" />
+    <PackageReference Include="Hackney.Shared.Person" Version="0.2.2-preview0004" />
```

Comment:
Seeing as we need to update the version, we shouldn't merge this PR until the version is updated